docs/guides/wifi.md

```diff
@@ -1,120 +1,60 @@
-| MacBookPro15,3   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,4   | BCM4377 | 4        | Formosa  | Big Sur              |
-| MacBookAir9,1    | BCM4377 | 4        | Fiji     | Big Sur              |
-| MacBookAir8,1    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacBookAir8,2    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacMini8,1       | BCM4364 | 3        | Lanai    | Mojave / Big Sur     |
-| MacPro7,1        | BCM4364 | 4        | Kahana   | Big Sur              |
-| iMac20,1         | BCM4364 | 4        | Hanauma  | Big Sur              |
-| iMac20,2         | BCM4364 | 4        | Kure     | Big Sur              |
-| iMac19,1*        | BCM4364 | 3        | Midway   | Mojave / Big Sur     |
-| iMac19,2*        | BCM4364 | 3        | Nihau    | Mojave / Big Sur     |
-| iMacPro1,1       | BCM4364 | ?        | Ekans?   | Mojave / Big Sur?    |
+   ```sh
+   cp -r /usr/share/firmware/wifi ~/Desktop
+   curl -L https://github.com/AsahiLinux/asahi-installer/archive/refs/heads/main.zip > ~/Desktop/asahi-installer-main.zip
+   ```
 
-\*iMac19,x don't have the T2 chip, but they have the same type of wifi chip as T2 Macs.
+   This shall make a folder named **wifi** and a zip archive named **asahi-installer-main.zip** on the macOS Desktop.
 
-*If there is missing/uncertain information for your model, please open a [pull request](https://github.com/t2linux/wiki/edit/master/docs/guides/wifi.md) or [issue](https://github.com/t2linux/wiki/issues/new) or mention it on the [discord](https://discord.com/invite/68MRhQu).*
+3. Copy the **wifi** folder and the **asahi-installer-main.zip** archive to an external drive or a partition which is formatted to a file system readable by Linux. You can also use your EFI partition by mounting it using `sudo diskutil mount disk0s1`.
 
-### Retrieving Firmware
+4. Boot into Linux.
 
-Run `ioreg -l | grep RequestedFiles` in macOS Terminal. Note down the output, and make sure you also know your Model Identifier. The output will be similar to this:
+5. Copy the **wifi** folder and the **asahi-installer-main.zip** archive to the **Home** folder using your file manager.
 
-```json
-"RequestedFiles" = ({
-    "Firmware"="C-4364s-B2/kauai.trx",
-    "TxCap"="C-4364s-B2/kauai-X3.txcb",
-    "Regulatory"="C-4364s-B2/kauai-X3.clmb",
-    "NVRAM"="C-4364s-B2/P-kauai-X3_M-HRPN_V-u__m-7.5.txt"
-})
-```
+6. Open the terminal and run :-
 
-You will next need to retrieve three of the four files listed by that command (the `.trx` "Firmware" file, the `.clmb` "Regulatory" file, and the `.txt` "NVRAM" file). The source of these files depends on which firmware type you will use. Copy them to somewhere that you can access them.
+   ```sh
+   unzip asahi-installer-main.zip
+   cd asahi-installer-main/src
+   python3 -m firmware.wifi ~/wifi firmware.tar
+   cd /lib/firmware
+   sudo tar xvf ~/asahi-installer-main/src/firmware.tar
+   sudo rm -r ~/asahi-installer-main* ~/wifi
```

Comment:
Not on Ubuntu

Reply:
cd /root required sudo

Reply:
> Not on Ubuntu

If sudo isn't needed in this line, then it shouldn't be there. Both files will be owned by the user, so root won't be required to remove them.

> cd /root required sudo

That would need sudo, but I'm not sure when `cd /root` is being used.

Reply:
> If sudo isn't needed in this line, then it shouldn't be there. Both files will be owned by the user, so root won't be required to remove them

Which files are you taking about?

Reply:
these ones `~/asahi-installer-main* ~/wifi`

Reply:
Without sudo, they seem to give an error stating that they are read-only on Ubuntu.

Reply:
I guess this was some security measure to prevent deletion in the home folder introduced by canonical (not sure though). Anyways, no harm is using sudo though.

Reply:
I think I misinterpreted what you meant by this

> Not on Ubuntu

but if that's the case then including sudo should be fine